### PR TITLE
Spanish api docs

### PR DESCRIPTION
**Pull request by baskof147**

### CODE REVIEW

File: affiliates/Spain.md [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+| first_name                      | Required <br /> must match regex: `/^[a-záéèíñóúüç[:space:]-\']+$/ui`                                                                         | First name of customer                                     |
+| last_name                       | Required <br /> must match regex: `/^[a-záéèíñóúüç[:space:]-\']+$/ui`                                                                         | Last name of the customer                                  |
+| second_last_name                | Required <br /> string (max 255), must match regex: `/^[a-záéèíñóúüç[:space:]-\']+$/ui`                                                       | Last name of the customer                                  |
+| nationality                     | Optional <br /> string (max 255), must match regex: `/^[a-záéèíñóúüç[:space:]-\']+$/ui`                                                       | Last name of the customer                                  |
```
Comment (mikk150) [inline affiliates/Spain.md]: nationality is last name of the customer?

File: affiliates/Spain.md [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+| phone_plan                      | Optional <br /> format: PREPAID, CONTRACT                                                                                                     | Phone plan of the customer                                 |
+| email                           | Required <br /> valid email address                                                                                                           | Email address                                              |
+| personal_id                     | Required <br /> must be valid PESEL                                                                                                                 | PESEL                                                      |
```
Comment (Nuffic) [inline affiliates/Spain.md]: PESEL = tidak valid

**Review by Nuffic — CHANGES_REQUESTED** [review]

File: affiliates/Spain.md (hunk 2 above)
Comment (baskof147) [reply affiliates/Spain.md]: fixed

File: affiliates/Spain.md [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+| phone_plan                      | Optional <br /> format: PREPAID, CONTRACT                                                                                                     | Phone plan of the customer                                 |
+| email                           | Required <br /> valid email address                                                                                                           | Email address                                              |
+| personal_id                     | Required <br /> must be valid DNI                                                                                                                   | DNI                                                      |
+| bank_account                    | Optional <br /> must be valid IBAN, must match regex: `/(ES|PT).*/`                                                                                 | Bank account                                               |
```
Comment (mikk150) [inline affiliates/Spain.md]: for some reason it renders incoreclty

**Review by mikk150 — CHANGES_REQUESTED** [review]

File: affiliates/Spain.md (hunk 1 above)
Comment (baskof147) [reply affiliates/Spain.md]: fixed

File: affiliates/Spain.md (hunk 3 above)
Comment (baskof147) [reply affiliates/Spain.md]: fixed

**Review by mikk150 — APPROVED** [review]

**Comment by Nuffic** [comment]
Pull request not actual, we are already moving to swagger docs.

**Pull request closed without merging**